src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -397,12 +399,52 @@ export class NfsFormComponent extends CdForm implements OnInit {
       }
       client.addresses = addressStr;
     });
-
     this.nfsForm.patchValue(res);
+    if (this.isEdit) {
+      this.resolveSubvolGrp(res);
+    }
     this.setPathValidation();
     this.clients = res.clients;
   }
 
+  resolveSubvolGrp(res: any) {
+    const splitPath = res.path.split('/');
+    const subvolumeGroup = splitPath[2];
+    const subvolume = splitPath[3];
+    const fs_name = this.nfsForm.getValue('fsal').fs_name;
+    this.subvolgrpService.get(fs_name).subscribe((data: CephfsSubvolumeGroup[]) => {
```

Comment:
Why do we need to pull this data again, aren't we already getting the data while we open the create or edit form?